cosmo_tester/framework/examples/hello_world.py

```diff
@@ -85,3 +85,16 @@ def assert_webserver_running(self, http_endpoint):
         if server_response.status_code != 200:
             pytest.fail('Unexpected status code: {}'.format(
                     server_response.status_code))
+
+
+def centos_hello_world(cfy, manager, attributes, ssh_key, tmpdir, logger,
+                       tenant='default_tenant', suffix=''):
+    hello = HelloWorldExample(
+        cfy, manager, attributes, ssh_key, logger, tmpdir,
+        tenant=tenant, suffix=suffix)
+    hello.blueprint_file = 'openstack-blueprint.yaml'
+    hello.inputs.update({
+        'agent_user': attributes.centos_7_username,
```

Comment:
this will need to be coordinated with #616

Reply:
Yeah, I'll have to merge one by one anyway, and adapt the changes as they come.